webpack.config.babel.js

```diff
@@ -0,0 +1,47 @@
+import path from 'path';
+import DashboardPlugin from 'webpack-dashboard/plugin';
+import HtmlWebpackPlugin from 'html-webpack-plugin';
+
+
+export default {
+    entry: './src/js/index.js',
+    output: {
+        filename: 'bundle.js',
+        path: path.resolve(__dirname, 'dist/build')
+    },
+    resolve: {
+        alias: {
+            '~': path.resolve(__dirname, 'src/js')
+        }
+    },
+    module: {
+        rules: [
+            {
+                test: /\.js$/,
+                exclude: /(node_modules|bower_components)/,
+                use: [
+                    {
+                        loader: 'babel-loader',
+                        options: {
+                            presets: ['env', 'react']
+                        }
+                    },
+                    {
+                        loader: 'eslint-loader'
+                    }
+                ]
+            },
+        ]
+    },
+    devServer: {
```

Comment:
After running `npm install yarn`, I ended up having to `brew install yarn` before being able to `yarn install`. Not sure why though...

But yeah I've got it up and running on `localhost:9000` and see that dashboard. Cool beanz.